### PR TITLE
json-rpc parameter bugfix

### PR DESCRIPTION
**Pull request by lidio601**
Following #128

### CODE REVIEW

File: lib/plugins/common/json-rpc.js [hunk 1]
```diff
@@ -9,11 +9,15 @@ module.exports = {fetch, parseConf}
 function fetch (account, method, params) {
   return Promise.resolve(true)
     .then(() => {
+      params = _.defaultTo({}, params)
```
[the remark this answers is not in the document]
Comment (lidio601) [reply lib/plugins/common/json-rpc.js]: uh, right. sorry I never know when I can use defaults

**Comment by lidio601** [comment]
all done

**Comment by lidio601** [comment]
https://trello.com/c/GmEpi71b/540-json-rpcaxios-error-cannot-read-property-url-of-undefined

**Comment by joshmh** [comment]
[json-rpc::axios error: Cannot read property 'url' of undefined](https://trello.com/c/GmEpi71b/540-json-rpcaxios-error-cannot-read-property-url-of-undefined)

**Pull request merged**